release-team/role-handbooks/enhancements/README.md

```diff
@@ -84,65 +95,87 @@ It is important that this process be followed and documentation remain up-to-dat
+    - Remove the milestone and change `tracked/yes` label to `tracked/no`
+- Clean up Enhancements issues by removing milestone from the enhancements that have not opted-in and make sure that number of in-tree open issues with current milestone matches number of opted-in enhancements.
+- Any enhancements removed from the milestone will now require an exception. As exception requests come in, discuss each with the Release Lead (and Shadows) to arrive at an approve/reject decision.
+    - Create an exception file in the Release for exceptions Example [1](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/exception.yaml).
+    - If a previously removed Enhancement has had their exception Approved, set their status to `Tracked` and use the `Enhancements` -> `Track Removed Enhancements` menu option to move it back to the `Enhancements` and `Docs` tabs.
 
 #### Post Enhancements Freeze (Week 5-10)
 
-- Stay on top of issues and continually monitor them twice a week and look at attached PRs. As Code Freeze gets closer, if there are PRs that have not been merged, move the issue to At Risk. If there is no activity, ping issue owners on either the issue or the k/k PR.
-- Monitor issues that are At Risk closely, almost daily. Code Freeze means no new code and keeping tabs on the status of the k/k PR is critical to planning. Make decisions if the enhancement should be deferred and work with SIG Leads to determine the best path forward.
+- Stay on top of issues and continually monitor them twice a week and look at attached PRs. As Code Freeze gets closer, if there are PRs that have not been merged, move the issue to `At Risk`. If there is no activity, ping issue owners on either the issue or the k/k PR.
+- Monitor issues that are `At Risk` closely, almost daily. Code Freeze means no new code and keeping tabs on the status of the k/k PR is critical to planning. Make decisions if the enhancement should be deferred and work with SIG Leads to determine the best path forward.
 
 #### Code Freeze (Week 10+)
 
-Start planning for the next release while assisting the Release Lead with anything relating to Analyst or Public Relation planning. Work with the Communications Lead to develop major themes for the official Kubernetes blog post.
+- Remove any enhancements that failed to merge their code by the Code freeze deadline.
+  - Set their status in the sheet to `Removed from Milestone`.
+  - Remove the milestone and change `tracked/yes` label to `tracked/no`.
+  - Remove the milestone from all open `k/k` PRs related to the enhancement.
+- Any enhancements removed from the milestone will now require an exception. As exception requests come in, discuss each with the Release Lead (and Shadows) to arrive at an approve/reject decision.
+  - Add a `/hold` label to all incoming exceptions to prevent from accidental merge.
```

Comment:
The exceptions come as an email unless we are changing the exceptions process. Where do we put the `/hold`?